autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -399,6 +399,13 @@ def fit(train_data,
                 Note: The file containing your `FeatureGenerator` class must be imported into current Python session in order to use a custom class.
             feature_generator_kwargs : dict, default={}
                 Keyword arguments to pass into the `FeatureGenerator` constructor.
+                Valid `AutoMLFeatureGenerator` kwargs:
+                    enable_nlp_vectorizer_features : bool, default = True
+                        If True, the vectorizer argument value is used to generate 'nlp_ngram' features from text features if present.
```

Comment:
Maybe add: "Try setting this to False if you encounter memory issues running AutoGluon and cannot access a machine with more memory".

Reply:
Added